src/components/originalSampleProcessing/blockProcessing/BlockProcessingLabwarePlan.tsx

```diff
@@ -176,7 +164,7 @@ const BlockProcessingLabwarePlan = React.forwardRef<HTMLDivElement, BlockProcess
                   name={`plans.${rowIndex}.replicateNumber`}
                   label={'Replicate Number'}
                   type={'text'}
-                  disabled={true}
+                  disabled={disableRepNumber}
```

Comment:
This can be replaced wiuth values.plans[rowIndex]?.replicateNumber

Reply:
Also, if you check in console this input is producing a react warning for changing from uncontrolled to controlled input which happens when an "undefined" value is passed to a component . This can be rectified by providing a value={values.plans[rowIndex] && values.plans[rowIndex].replicateNumber?values.plans[rowIndex].replicateNumber
                      : ''
                  }